apps/server/src/modules/learnroom/service/common-cartridge-export.service.ts

```diff
@@ -104,10 +104,21 @@ export class CommonCartridgeExportService {
 		for await (const columnBoardId of columnBoardIds) {
 			const columnBoard = await this.columnBoardService.findById(columnBoardId);
 
-			builder.addOrganization({
+			const organization = builder.addOrganization({
 				title: columnBoard.title,
 				identifier: createIdentifier(columnBoard.id),
 			});
+
+			if (columnBoard.children.length > 0) {
+				columnBoard.children
+					.filter((child) => child instanceof Column)
+					.map((column) =>
+						organization.addSubOrganization({
+							title: (column as Column).title,
+							identifier: createIdentifier((column as Column).id),
+						})
+					);
+			}
```

Comment:
- If statement is not required
- map -> forEach
- type cast is not necessary for id